### PR TITLE
{bio}[foss/2015b] libff1 3.2.1, gettext 0.19.7, GLib 2.46.2, Python 2.7.9, Exonerate 2.4.0 (REVIEW)

### PR DESCRIPTION
**Pull request by nathanhaigh**
Working on Easyconfigs for Exonerate-2.4.0-foss-2015b

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
missing easyconfig for `GLib/2.46.2-foss-2015b`

**Comment by nathanhaigh** [comment]
Test report by @nathanhaigh
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux ubuntu 14.04, Intel(R) Xeon(R) CPU E5-2699 v3 @ 2.30GHz, Python 2.7.6
See https://gist.github.com/140a0d2646cdba3d9fb6 for a full test report.

**Comment by nathanhaigh** [comment]
Test report by @nathanhaigh
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux ubuntu 14.04, Intel(R) Xeon(R) CPU E5-2699 v3 @ 2.30GHz, Python 2.7.6
See https://gist.github.com/c19c8cb0cadf91bd8682 for a full test report.

File: easybuild/easyconfigs/g/GLib/GLib-2.46.2-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://ftp.gnome.org/pub/gnome/sources/glib/%(version_major_minor)s/']
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('libffi', '3.2.1'),
+    ('gettext', '0.19.7'),
+]
+#builddependencies = [('Python', '2.7.9')]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GLib/GLib-2.46.2-foss-2015b.eb]: this should be there (some version)?

**Comment by boegel** [comment]
@nathanhaigh: Jenkins doesn't agree with you

```
Irresolvable dependencies encountered: SQLite/3.8.8.1-foss-2015b, Tk/8.6.3-foss-2015b-no-X11
```

**Comment by boegel** [comment]
@nathanhaigh: one more...

```
Irresolvable dependencies encountered: Tcl/8.6.3-foss-2015b
```

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 5 out of 8 (8 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e8036cbd553163d9e5e6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2090.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/79f38dec46cc3fdc1a58 for a full test report.

**Comment by boegel** [comment]
@nathanhaigh: gettext also needs libxml2?

**Comment by nathanhaigh** [comment]
@boegel indeed, all the `gettext` easyconfigs look like they are missing deps: #2652

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2090.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c4ce25138f46e3ed120b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @nathanhaigh!

**Pull request merged**